Examples/Examples/WorldScaleExampleView.swift

```diff
@@ -51,7 +51,6 @@ struct WorldScaleGeoTrackingExampleView: View {
                     }
                 }
         }
-        .calibrationViewHidden(false)
         .calibrationViewAlignment(.bottomLeading)
         .task {
```

Comment:
The locationDataSource should be started in this task block otherwise the circle graphic around the initial location does not show.